### PR TITLE
Added method to clear all points in Line2D

### PR DESCRIPTION
**Pull request by Chaosus**
I found this operation very useful when working with this class and created method for this.

### CODE REVIEW

**Comment by reduz** [comment]
seems fine

File: doc/classes/Line2D.xml [hunk 1]
```diff
@@ -20,6 +20,13 @@
 				Add a point at the [code]position[/code]. Appends the point at the end of the line.
 			</description>
 		</method>
+		<method name="clear_points">
+			<return type="void">
+			</return>
+			<description>
+				Removes all points from the line.
```
Comment (cbscribe) [inline doc/classes/Line2D.xml]: Looks good!

**Comment by akien-mga** [comment]
Thanks!

**Comment by hpvb** [comment]
Cherry-picked for 3.1.1.

**Pull request merged**